lib/capybara/apparition/driver.rb

```diff
@@ -66,7 +66,9 @@ def client
       @client ||= begin
         @launcher ||= Browser::Launcher.start(options)
         ws_url = @launcher.ws_url
-        ::Capybara::Apparition::ChromeClient.client(ws_url.to_s)
+        client = ::Capybara::Apparition::ChromeClient.client(ws_url.to_s)
+        client.timeout = session_wait_time
+        client
```

Comment:
A possible simplification:
```suggestion
        ::Capybara::Apparition::ChromeClient.client(ws_url.to_s).tap { |client| client.timeout = session_wait_time }
```